unit_tests/test_version_utils.py

```diff
@@ -144,39 +144,36 @@ def test_07_get_all_versions(self):
     def test_08_assume_versions_oss(self):
         with unittest.mock.patch.object(os.environ, 'get', return_value='branch-5.0', clear=True):
             params = {}
-            is_version_enterprise, version = assume_version(params)
-            self.assertTrue(not is_version_enterprise, 'This should be OSS')
+            version = assume_version(params)
             self.assertEqual(version, 'nightly-5.0', 'Version should be 5.0')
 
             scylla_version = '5.0'
-            is_version_enterprise, version = assume_version(params, scylla_version)
-            self.assertTrue(not is_version_enterprise, 'This should be OSS')
+            version = assume_version(params, scylla_version)
             self.assertEqual(version, 'nightly-5.0', 'Version should be 5.0')
 
             repo_url = 'https://s3.amazonaws.com/downloads.scylladb.com/unstable/scylla/branch-5.0/rpm/centos/' \
                        '2022-05-10T07:40:41Z/scylla.repo'
             params.update({'scylla_repo': repo_url})
-            is_version_enterprise, version = assume_version(params)
-            self.assertTrue(not is_version_enterprise, 'This should be OSS')
+            version = assume_version(params)
             self.assertEqual(version, 'nightly-5.0', 'Version should be 5.0')
 
     def test_09_assume_versions_enterprise(self):
```

Comment:
This test and above one duplicate each other.
So, need to do following changes:
- remove above one (`test_08_assume_versions_oss`)
- rename this one removing the `_enterprise` suffix.
- remove useless statement in this test such as `self.assertTrue('This should be enterprise')`

Reply:
Done